doc/modules/partial_dependence.rst

```diff
@@ -1,21 +1,25 @@
 
 .. _partial_dependence:
 
-========================
-Partial dependence plots
-========================
+==================
+Partial dependence
+==================
 
 .. currentmodule:: sklearn.inspection
 
-Partial dependence plots (PDP) show the dependence between the target
+Partial dependence plots (PDP) and individual conditional
```

Comment:
I really think that we should separate both approaches. Maybe, it will make sense to add first ICE and then PDP and explain that this the average response. I fell that we might miss a small "mathematical formulation" section. It could be a good addition